lighthouse-core/config/legacy-default-config.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright 2018 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+import defaultConfig from './default-config.js';
+
+const unsupportedAuditIds = [
+  'experimental-interaction-to-next-paint',
+  'uses-responsive-images-snapshot',
+  'work-during-interaction',
+];
+
+const audits = defaultConfig.audits?.filter(audit =>
+  !unsupportedAuditIds.find(auditId => audit.toString().endsWith(auditId)));
+
+/** @type {LH.Config.Category} */
+const performance = {
+  // @ts-expect-error categories will always exist on the default config.
+  ...defaultConfig.categories['performance'],
+  auditRefs: defaultConfig.categories?.['performance'].auditRefs
+    .filter(auditRef => !unsupportedAuditIds.includes(auditRef.id)) || [],
+};
+
+/** @type {Record<string, LH.Config.Category>} */
+const categories = {
+  ...defaultConfig.categories,
+  'performance': performance,
+};
```

Comment:
This bit is confusing, a comment would help.

`Remove unsupported audits from the performance category` (alt: do a JSON.stringify copy and modify in place)

and a comment at top of file

`Construct legacy navigation config`